File: app/src/test/java/io/github/tobyhs/weatherweight/data/LocationNotFoundErrorTest.java
package io.github.tobyhs.weatherweight.data;

import org.junit.Test;

import static org.hamcrest.CoreMatchers.is;
import static org.hamcrest.MatcherAssert.assertThat;

public class LocationNotFoundErrorTest {
    @Test
    public void constructorSetsMessage() {
        LocationNotFoundError error = new LocationNotFoundError("Nowhere");
        assertThat(error.getMessage(), is("Location Nowhere not found"));
    }
}

File: app/src/test/java/io/github/tobyhs/weatherweight/data/LocationNotFoundErrorTest.kt
package io.github.tobyhs.weatherweight.data

import org.hamcrest.CoreMatchers.equalTo
import org.hamcrest.MatcherAssert.assertThat

import org.junit.Test

class LocationNotFoundErrorTest {
    @Test
    fun constructorSetsMessage() {
        val error = LocationNotFoundError("Nowhere")
        assertThat(error.message, equalTo("Location Nowhere not found"))
    }
}
